homeassistant/components/bthome/__init__.py

```diff
@@ -55,6 +55,7 @@ def process_service_info(
             sensor_device_info = update.devices[device_key.device_id]
             device = device_registry.async_get_or_create(
                 config_entry_id=entry.entry_id,
+                connections={(BLUETOOTH_DOMAIN, address)},
```

Comment:
Shouldn't it be `CONNECTION_BLUETOOTH` as Frenck showed in his screenshot?

Reply:
Yes, that's right.